patterns/dswp-secondary-hero-image-with-title.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Title: DSWP Secondary Hero Image With Title
+ * Slug: design-system-wordpress-theme/dswp-secondary-hero-image-with-title
```

Comment:
sorry, if this was already discuss, but just making sure there is a naming convention we are using, if so, seems like there might be some duplication or unnecessary naming for example if we have a name space of `design-system-wordpress-theme` do we need the name of the pattern starting off with `dswp-`